src/libraries/restore/runtime/runtime.depproj

```diff
@@ -28,7 +28,7 @@
 
   <ItemGroup>
     <!-- System.Data.SqlClient is not live-built anymore -->
-    <PackageReference Include="System.Data.SqlClient" Version="$(SystemDataSqlClientVersion)" />
+    <PackageReference Condition="'$(BinPlaceTestRuntimePack)' != 'true'" Include="System.Data.SqlClient" Version="$(SystemDataSqlClientVersion)" />
```

Comment:
Merge this ItemGroup with the ItemGroup above and share the condition as suggested above.

Reply:
Thanks!